overlay/etc/systemd/system/sync-ssh-key.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Sync ssh key for MCO
+# Removal of this file signals firstboot completion
+ConditionPathExists=!/etc/ignition-machine-config-encapsulated.json
```

Comment:
to be extra-safe, should we add
```
ConditionPathExists=/home/core/.ssh/authorized_keys.d/ignition
```
?

Reply:
and maybe also:
```
ConditionPathExists=!/home/core/.ssh/authorized_keys
```

Reply:
Good idea, lets try it

Reply:
since we want this to run at first boot, shouldn't this be 

`ConditionPathExists=/etc/ignition-machine-config-encapsulated.json`

Reply:
or do I have it backwards?